### ISSUE TITLE
Please can you help and advise how to remove the random line....

### ISSUE THREAD

**Issue by Matthew2727** [kylegilman/video-embed-thumbnail-generator#287]
Hi Kyle,

Hats off for your development wok on this great plugin. Well done you and holding down a career elsewhere and doing this on the side is quite something.

When you get a minute.... Take a look at my screenshot. For some reason when viewed through most browsers for example Chrome (especially mobile and tablet browsers) I get a line above the video. I can live with it but if you knew a way to remove that would be fab.

Its on www.contentcaptureservices.co.uk on the homepage. I designed the video on Moovly. I think its an mp4.

I know that you are busy but any advice would be appreciated.

Thanks,
Matthew.
![line above video](https://cloud.githubusercontent.com/assets/17214504/13026683/bce8795e-d228-11e5-9f65-63051fe841fc.jpg)

**Comment by kylegilman** [comment]
Hi Matthew, your site looks great and I can see how that black line would really annoy you. On first load in all of my browsers in OS X the line doesn't show up, but I was able to force it to appear by resizing the window just right. I think it's a rounding error when the video is scaled that makes a portion of the thumbnail element's background color show instead of the thumbnail image. I was able to fix it by adding this custom CSS
```
.video-js .vjs-poster {
    background-color: white;
}
```
The easiest way to add that to your site is to use the Custom CSS Jetpack module.
